Sources/NSPredicate+Sweetness.swift

```diff
@@ -175,4 +176,36 @@ extension NSPredicate {
             lessThanOrEqualToSecond
         ])
     }
+    
+    public static func isValue<Root, Property: ComparableProperty>(for keyPath: KeyPath<Root, Property>,
+                                                                   in array: [Property]) -> NSPredicate {
+        return NSPredicate(format: "%K IN %@", argumentArray: [
```

Comment:
would there be any value in adding NOT IN ? 😄

Reply:
ooh good point